test/tests/pbs_Rrecord_resources_used.py

```diff
@@ -183,98 +188,121 @@ def common(self, is_nonrerunnable, restart_mom):
         """
         a = {ATTR_JobHistoryEnable: 1}
         self.server.manager(MGR_CMD_SET, SERVER, a)
 
-        jid1,jid2,jid3s1 = self.common(True,'r')
+        jid1, jid2, jid3s1 = self.common(True, 'r')
 
         m = self.server.accounting_match(
-        msg='.*R;' + jid1 + '.*resources_used.*run_count=1', id=jid1, regexp=True)
+            msg='.*R;' + jid1 + '.*resources_used.*run_count=1', id=jid1,
+            regexp=True)
         self.assertNotEqual(m, None)
         m = self.server.accounting_match(
-        msg='.*R;' + jid2 + '.*resources_used.*run_count=1', id=jid2, regexp=True)
+            msg='.*R;' + jid2 + '.*resources_used.*run_count=1', id=jid2,
+            regexp=True)
         self.assertEqual(m, None)
-        # Use re.replace() as accounting_match cannot match subjobs []
         m = self.server.accounting_match(
-        msg='.*R;' + re.escape(jid3s1) + '.*resources_used.*run_count=1', id=jid3s1, regexp=True)
+            msg='.*R;' + re.escape(jid3s1) + '.*resources_used.*run_count=1',
+            id=jid3s1, regexp=True)
         self.assertNotEqual(m, None)
+        self.server.manager(MGR_CMD_SET, SERVER, {'scheduling': 'True'},
+                            expect=True)
 
     def test_t4(self):
         """
         Scenario: Mom restarted without '-r' option and requeued using qrerun.
-        Expected outcome: Server should record last known resource usage in the 'R' record for both.
+        Expected outcome: Server should record last known resource usage in
+                  the 'R' record for both.
         """
-        jid1,jid2,jid3s1 = self.common(False,'s')
+        jid1, jid2, jid3s1 = self.common(False, 's')
 
         m = self.server.accounting_match(
-        msg='.*R;' + jid1 + '.*resources_used.*run_count=1', id=jid1, regexp=True)
+            msg='.*R;' + jid1 + '.*resources_used.*run_count=1', id=jid1,
+            regexp=True)
         self.assertNotEqual(m, None)
         m = self.server.accounting_match(
-        msg='.*R;' + jid2 + '.*resources_used.*run_count=1', id=jid2, regexp=True)
+            msg='.*R;' + jid2 + '.*resources_used.*run_count=1', id=jid2,
+            regexp=True)
         self.assertNotEqual(m, None)
-        # Use re.replace() as accounting_match cannot match subjobs []
         m = self.server.accounting_match(
-        msg='.*R;' + re.escape(jid3s1) + '.*resources_used.*run_count=1', id=jid3s1, regexp=True)
+            msg='.*R;' + re.escape(jid3s1) + '.*resources_used.*run_count=1',
+            id=jid3s1, regexp=True)
         self.assertNotEqual(m, None)
         """
         Verify that the jobs are in 'Q' state.
         """
-        self.server.expect(JOB, {ATTR_state: 'Q'}, jid1, offset=5)
+        self.server.expect(JOB, {ATTR_state: 'Q'}, jid1)
         self.server.expect(JOB, {ATTR_state: 'Q'}, jid2)
         self.server.expect(JOB, {ATTR_state: 'Q'}, jid3s1)
+
+        self.server.manager(MGR_CMD_SET, SERVER, {'scheduling': 'True'},
```

Comment:
No, it will not raise an exception. We can remove it. However, there are many places, where I see the usage being similar to how I have used it.
Please let me know if you want me to remove it.